### DISCUSSION TITLE
Why am I getting this warning on every notebook line following medspacy.load()?

### DISCUSSION THREAD

**Discussion by RonWilkinson** [medspacy/medspacy#64] [Q&A]
I have just installed medspacy and am trying out the notebook examples.  Once I run medspacy.load(), the following warning appears following that command and any command whatever that I give in subsequent cells:

> C:\Users\Ron\anaconda3\lib\site-packages\ipykernel\ipkernel.py:287: DeprecationWarning: `should_run_async` will not call `transform_cell` automatically in the future. Please pass the result to `transformed_cell` argument and any exception that happen during thetransform in `preprocessing_exc_tuple` in IPython 7.17 and above.
  and should_run_async(code)

Can anyone tell me what is going on and how to fix it?

**Comment by abchapman93** [comment]
This warning is an issue with IPython that you can fix by either upgrading or downgrading your IPython version. The solutions offered here should fix it: https://stackoverflow.com/questions/63413807/deprecation-warning-from-jupyter-should-run-async-will-not-call-transform-c
